src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -129,6 +130,17 @@ public void clear() {
         internalList.clear();
     }
 
+    /**
+     * Sorts all persons in list.
+     */
+    public void sort() {
+        Collections.sort(internalList, new Comparator<Person>(){
+            public int compare(Person p1, Person p2) {
+                return p1.getName().toString().compareTo(p2.getName().toString());
+            }
```

Comment:
maybe consider making Person comparable? so we can avoid the comparator for every single sort.